frontend/src/scenes/dashboard/dashboardLogic.tsx

```diff
@@ -695,16 +695,38 @@ export const dashboardLogic = kea<dashboardLogicType>([
                 return sortedDates[0]
             },
         ],
+        sortedClientRefreshAllowed: [
```

Comment:
This could be rolled into the other selector below, since we're not really using the sorted times elsewhere.